grammars/julia.cson

```diff
@@ -146,7 +148,7 @@ repository:
             name: "entity.name.function.julia"
           "3":
             name: "support.type.julia"
-        match: "\\b(function|macro)\\s+([\\w.!]+)({[^}]*})?\\("
+        match: "\\b(function|macro)\\s+([\\w.!]+)({(?:[^{}]|{(?:[^{}]|{[^{}]*})*})*})?\\("
```

Comment:
I won't be in front of the code for a bit -- if it's the function
declarations, I'd go either way -- no strong opinion -- without
highlighting the module seems more consistent, but having it higlighted
that you're extending another module is a nice touch, too.

On Wed, Jul 8, 2015 at 5:23 PM, Spencer Lyon <notifications@github.com>
wrote:

> In grammars/julia.cson
> <https://github.com/JuliaLang/atom-language-julia/pull/19#discussion_r34209960>
> :
>
> > @@ -146,7 +148,7 @@ repository:
> >              name: "entity.name.function.julia"
> >            "3":
> >              name: "support.type.julia"
> > -        match: "\\b(function|macro)\\s+([\\w.!]+)({[^}]*})?\\("
> > +        match: "\\b(function|macro)\\s+([\\w.!]+)({(?:[^{}]|{(?:[^{}]|{[^{}]*})*})*})?\\("
>
> Should we drop the . here also?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaLang/atom-language-julia/pull/19/files#r34209960>
> .
>